tests/core/mountpoint/test_folder_operations.py

```diff
@@ -213,7 +213,11 @@ def move_folder(self, src, dst_parent, dst_name):
         def iterdir(self, path):
             expected_exc = None
             try:
-                expected_children = {x.name for x in path.to_oracle().iterdir()}
+                expected_children = {
+                    x.name
+                    for x in path.to_oracle().iterdir()
```

Comment:
are you sure the filter should be done on the oracle ?

I would say the `.fuse_hidden` files appear in the fuse mountpoint, hence in the thing we are testing (the oracle is a regular folder that is suppose to provide us the expected behavior the mountpoint should have)